<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendCoinsEntry</class>
 <widget class="QStackedWidget" name="SendCoinsEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>182</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="SendCoins">
   <property name="toolTip">
    <string>This is a normal payment.</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <property name="horizontalSpacing">
     <number>12</number>
    </property>
    <property name="verticalSpacing">
     <number>8</number>
    </property>
    <item row="1" column="1">
     <widget class="QLineEdit" name="addAsLabel">
      <property name="toolTip">
       <string>Enter a label for this address to add it to the list of used addresses</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="messageLabel">
      <property name="text">
       <string>Message:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="messageTextLabel">
      <property name="toolTip">
       <string>A message that was attached to the raptoreum: URI which will be stored with the transaction for your reference. Note: This message will not be sent over the Raptoreum network.</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="payToLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QValidatedLineEdit" name="payTo">
        <property name="toolTip">
         <string>The Raptoreum address to send the payment to</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="addressBookButton">
        <property name="toolTip">
         <string>Choose previously used address</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Alt+A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pasteButton">
        <property name="toolTip">
         <string>Paste address from clipboard</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Alt+P</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteButton">
        <property name="toolTip">
         <string>Remove this entry</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="payToLabel">
      <property name="text">
       <string>Pay &amp;To:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>payTo</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayoutAmount" stretch="0,0,0,0">
      <item>
       <widget class="BitcoinAmountField" name="payAmount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxSubtractFeeFromAmount">
        <property name="toolTip">
         <string>The fee will be deducted from the amount being sent. The recipient will receive a lower amount of Raptoreum than you enter in the amount field. If multiple recipients are selected, the fee is split equally.</string>
        </property>
        <property name="text">
         <string>S&amp;ubtract fee from amount</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="futureCb">
        <property name="toolTip">
         <string>Enable/Disable future transaction</string>
        </property>
        <property name="text">
         <string>future</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="useAvailableBalanceButton">
        <property name="text">
         <string>Use available balance</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labellLabel">
      <property name="text">
       <string>&amp;Label:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>addAsLabel</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="maturityLb">
      <property name="text">
       <string>Maturity:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="amountLabel">
      <property name="text">
       <string>A&amp;mount:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>payAmount</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="futureHorizontalLayout">
      <item>
       <widget class="QLineEdit" name="maturity">
        <property name="toolTip">
         <string>Number of block confirmations for this output to be spendable if happen before timelock</string>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="locktimeLb">
        <property name="text">
         <string>Lock time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="locktime">
        <property name="toolTip">
         <string>Time in seconds for this output to be locked if happen before maturity</string>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="feeLb">
        <property name="text">
         <string>Fee:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="feeDisplay">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="SendCoins_UnauthenticatedPaymentRequest">
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="toolTip">
    <string>This is an unauthenticated payment request.</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_is">
    <property name="spacing">
     <number>12</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="payToLabel_is">
      <property name="text">
       <string>Pay To:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="payToLayout_is">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="payTo_is"/>
      </item>
      <item>
       <widget class="QToolButton" name="deleteButton_is">
        <property name="toolTip">
         <string>Remove this entry</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="memoLabel_is">
      <property name="text">
       <string>Memo:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="memoTextLabel_is">
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="amountLabel_is">
      <property name="text">
       <string>A&amp;mount:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>payAmount_is</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="BitcoinAmountField" name="payAmount_is">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="SendCoins_AuthenticatedPaymentRequest">
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="toolTip">
    <string>This is an authenticated payment request.</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_s">
    <property name="spacing">
     <number>12</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="payToLabel_s">
      <property name="text">
       <string>Pay To:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="payToLayout_s">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="payTo_s">
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteButton_s">
        <property name="toolTip">
         <string>Remove this entry</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="memoLabel_s">
      <property name="text">
       <string>Memo:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="memoTextLabel_s">
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="amountLabel_s">
      <property name="text">
       <string>A&amp;mount:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>payAmount_s</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="BitcoinAmountField" name="payAmount_s">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qt/qvalidatedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>qt/bitcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>payTo</tabstop>
  <tabstop>addressBookButton</tabstop>
  <tabstop>pasteButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>addAsLabel</tabstop>
  <tabstop>payAmount</tabstop>
  <tabstop>payAmount_is</tabstop>
  <tabstop>deleteButton_is</tabstop>
  <tabstop>payAmount_s</tabstop>
  <tabstop>deleteButton_s</tabstop>
 </tabstops>
 <resources>
  <include location="../raptoreum.qrc"/>
 </resources>
 <connections/>
</ui>
